bug(core): Compiling core on Linux outputs a warning

Compiling core outputs the following warning when building on Linux:

```
core/include/keyman/keyboardprocessor.h:130:1: warning: ‘visibility’ attribute ignored [-Wattributes]
  130 | typedef KMN_API uint8_t (*km_kbp_keyboard_imx_platform)(km_kbp_state*, uint32_t, void*);
      | ^~~~~~~
```

--- a/core/include/keyman/keyboardprocessor.h
+++ b/core/include/keyman/keyboardprocessor.h
@@ -127,7 +127,7 @@ typedef struct km_kbp_option_item  km_kbp_option_item;
 // Callback function used to to access Input Method eXtension library functions
 // from Keyman Core
 //
-typedef KMN_API uint8_t (*km_kbp_keyboard_imx_platform)(km_kbp_state*, uint32_t, void*);
+typedef uint8_t (*km_kbp_keyboard_imx_platform)(km_kbp_state*, uint32_t, void*);
 
 /*```
 ### Error Handling

